tests/unit/utils/sparqlhandler/test_data_cube_inspector.py

```diff
@@ -180,62 +225,6 @@ def test_get_dsd_qube_components_for_csv():
         True,
     )
 
-    component = get_dsd_component_by_property_url(
```

Comment:
Why has this section disappered?

Reply:
I believe we still test this function in this file, but now we do it a bit more thoroughly by testing it on a standard shape input, single measure pivoted, multi measure pivoted inputs. That's why it looks like this test has disappeared. If this is unnecessary then I can undo the change and only have a single test.

Reply:
This unit test is gone because it has been split into three different tests now, testing the get_dsd_component_by_property_url and get_dsd_qube_components_for_csv functions with a different input (e.g. standard shape data cube, pivoted shape single measure, pivoted shape multi measure). I believe we received this change from the merge with main.